Sources/MongoSwift/ConnectionPool.swift

```diff
@@ -98,7 +98,7 @@ internal class ConnectionPool {
     }
 
     /// Temporary API for testing purposes. This will likely change in SWIFT-471.
-    internal func setSSLOpts(caFile: String?, pemFile: String?) throws {
+    internal func setSSLOpts(caFile: String?, pemFile: String?, allowInvalidHostnames: Bool? = nil) throws {
```

Comment:
adding the capability to set this was necessary because the hosts in the test URIs do not match up with the MO replica sets.